nextflow.config

```diff
@@ -40,6 +40,11 @@ params {
     // Cellranger parameters
     cellranger_index    = null
 
+    // Cellranger ARC parameters
+    cellrangerarc_index = null
+    motifs              = null
+    sample_type         = 'gex'
```

Comment:
A question here since I have limited knowledge on the data, but can opinate on the code.
If we have this as a parameter, it means the samplesheet will be completely of that specific sample type, not allowing users to have mixed data in their samplesheet and being able to analyse everything at the same time.

The question here is, is this the desired behaviour? Or for example, would be ok/nice to allow users to allow all at once provided that they give the correct sample_type value in the samplesheet?

Don't know if that would be an used case though, is just a general question.

Reply:
May the parameter is a bit confusing how I implemented it. I set the defualt sample type as gex because one might just simply analyse single cell RNA seq data. 

To explain. We would now cover two possible scenarios: 

single cell RNA --> user does not have to specify sample type in the sample sheet
multiome scRNA + scATAC --> user has to specify the sample type 'gex' for scRNA datasets and 'atac' for scATAC datasets (both he must provide in the samplesheet)

in the future (this subworkflow I have also ready):
singe cell ATAC --> user has to specify the sample type atac in the sample sheet

Reply:
Another possibility would be to cover that in the samplesheet validation. If `sample_type` is missing, or null, then `gex` is assumed for that particular sample. 

As it's a per-sample setting, I find it a bit confusing to have a global parameter for this.

Reply:
I changed it accroding to grst recommendation.